solc/CommandLineParser.cpp

```diff
@@ -828,7 +828,7 @@ General Information)").c_str(),
 		)
 		(
 			g_strModelCheckerSolvers.c_str(),
-			po::value<string>()->value_name("all,cvc4,z3,smtlib2")->default_value("all"),
+			po::value<string>()->value_name("cvc4,eld,z3,smtlib2")->default_value("z3"),
```

Comment:
If we keep it at `all`, we always get warnings saying some solver was enabled but is not available. Better to change the default to `z3`.

Reply:
Sounds reasonable.